realsense2_camera/test/utils/pytest_rs_utils.py

```diff
@@ -833,7 +837,13 @@ def set_bool_param(self, param_name, param_value):
         new_param_value = ParameterValue(type=ParameterType.PARAMETER_BOOL, bool_value=param_value)
         req.parameters = [Parameter(name=param_name, value=new_param_value)]
         return self.send_param(req)
-
+    
+    def get_bool_param(self, param_name):
+        req = GetParameters.Request()
+        req.names = [param_name]
+        value = self.get_param(req)
```

Comment:
please add a check to verify whether the response is not None.

Reply:
fixed